src/pretix/presale/templates/pretixpresale/event/fragment_addon_choice.html

```diff
@@ -127,28 +127,29 @@ <h5 id="cp-{{ form.pos.pk }}-item-{{ item.pk }}-{{ var.pk }}-legend">{{ var }}</
-                                                    title="{% blocktrans trimmed with item=var.value %}Modify price for {{ item }}{% endblocktrans %}"
-                                                    step="any"
-                                                    value="{% if event.settings.display_net_prices %}{{ var.initial_price.net|money_numberfield:event.currency }}{% else %}{{ var.initial_price.gross|money_numberfield:event.currency }}{% endif %}"
-                                            >
-                                        </div>
-                                    {% elif not var.display_price.gross %}
-                                    {% elif event.settings.display_net_prices %}
-                                        {{ var.display_price.net|money:event.currency }}
-                                    {% else %}
-                                        {{ var.display_price.gross|money:event.currency }}
-                                    {% endif %}
-                                    {% if item.original_price or var.original_price %}
-                                        </ins>
-                                    {% endif %}
+                                        {% if item.free_price %}
+                                            <div class="input-group input-group-price">
+                                                <span class="input-group-addon">{{ event.currency }}</span>
+                                                <input type="number" class="form-control input-item-price"
+                                                        id="price-variation-{{form.pos.pk}}-{{ item.pk }}-{{ var.pk }}"
+                                                        placeholder="0"
+                                                        min="{% if event.settings.display_net_prices %}{{ var.display_price.net|money_numberfield:event.currency }}{% else %}{{ var.display_price.gross|money_numberfield:event.currency }}{% endif %}"
+                                                        name="cp_{{ form.pos.pk }}_variation_{{ item.id }}_{{ var.id }}_price"
+                                                        title="{% blocktrans trimmed with item=var.value %}Modify price for {{ item }}{% endblocktrans %}"
+                                                        step="any"
+                                                        value="{% if event.settings.display_net_prices %}{{ var.initial_price.net|money_numberfield:event.currency }}{% else %}{{ var.initial_price.gross|money_numberfield:event.currency }}{% endif %}"
+                                                >
+                                            </div>
+                                        {% elif not var.display_price.gross %}
+                                            <span class="text-uppercase">{% trans "free" context "price" %}</span>
```

Comment:
All the changes in this block around that line are just indentation fixes, this line is the only actual addition